extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/health/GrpcHealthEndpoint.java

```diff
@@ -10,14 +10,12 @@
 import grpc.health.v1.HealthOuterClass.HealthCheckResponse.ServingStatus;
 import grpc.health.v1.MutinyHealthGrpc;
 import io.quarkus.grpc.GrpcService;
-import io.quarkus.grpc.runtime.supports.context.GrpcEnableRequestContext;
 import io.smallrye.mutiny.Multi;
 import io.smallrye.mutiny.Uni;
 import io.smallrye.mutiny.operators.multi.processors.BroadcastProcessor;
 
 // Note that we need to add the scope and interceptor binding explicitly because this class is not part of the index
```

Comment:
Should remove "and interceptor binding" now that `@GrpcEnableRequestContext` is gone?